internal/app/apiserver/apiserver.go

```diff
@@ -0,0 +1,65 @@
+package apiserver
+
+import (
+	"encoding/json"
+	"github.com/gorilla/mux"
+	"net/http"
+)
+
+type Input struct{
+	Word string `json:"word"`
+	Source string `json:"source"`
+	Target string `json:"target"`
+}
+type Response struct {
+	TranslatedWord string
+}
+
+
+type Server struct {
+	*mux.Router
+
+}
+
+func NewServer() *Server{
+	s := &Server{
+		Router: mux.NewRouter(),
+	}
+	s.routes()
+	return s
+}
+
+func (s *Server) routes(){
```

Comment:
Routing and route handlers should be in separate package. Elements of the application should be decoupled as much as possible.